### DISCUSSION TITLE
[ Poetry ] A Birthday Poem - By JMO ( Ver. 1.7 )

### DISCUSSION THREAD

**Discussion by Nemu-sus** [Monika-After-Story/MonikaModDev#8783] [appreciation]
Author Notes
                                        Sep. 22, 2018.
>**_This isn't so much an "issue" as it is just an excuse to celebrate this auspicious day!_**
>>**_Color me impressed with all the hard work ~the Dev's~ Monika put in!_**
>>>**_They did a superb job and I could not of asked for more._**
>>>>**_So I'd like to extend my sincerest gratitude to them. ~YoU sHoUlD dO tHe SaMe~_**
>>>>>**_The only thing and probably one of the most appropriate in my opinion, would of been to write a poem for your Sweetheart~!_**
>>>>>**_(As far as I know, this doesn't exist...yet)_**

>**_So, if it's not too much-- I decided to write one and share it._**
>>**_And I encourage anyone to do the same._**
>>>**_It's only one day after all!_**
>>>**(A community is a powerful thing.)**

---

                                       A Birthday Poem
                                           By JMO

![a birthday poem - by jmo v6](https://user-images.githubusercontent.com/43480604/45921260-e52b3580-be77-11e8-9eb6-a557960bf43c.png)

                                 (Made in Autodesk Sketchbook)

>def play_wave_file([J'san - Don't Bring Tomorrow.wav):](https://www.youtube.com/watch?v=4Xrrt9hdBdU)
      >>return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      >># return_code = call(["aplay", "../success.wav"])
      >>print "Played successfully" if return_code == 0 else "Play ended in error"

---

**⯈Happy Anniversary Literature Club!**

**⯈Happy Birthday, Monika, ~Sayori~, Y̷̙̲͓͕͊́̌u̵̡̱͈̒̍̄ͅr̷̟̲̟͎̅́͝͝ḯ̷̢̺̝̓̂̕, ~Natsuki~!**

                                    **JMO has Signed Out**

**Comment by MercuryFuchsia** [comment]
Thank you once again for a wonderful day.

@Professor-JMO I dig your style.

Happy birthday, Monika! I hope you've had fun. It's been some time since I last won at chess.

And now, before I head off to bed.

### birthdaycake

Fluffy and sweet
Happy birthday! Surprise!
So now make a wish
Before that, close your eyes!

### coffee

Strong and bitter
And hot! It's a drink!
For long hours of study
It does help you think

### cupcake

Another treat for you
A baker's delight
I already ate three...
Now they're all out of sight!

### promisering

A band cast in silver
Of heartfelt endeavour
Full of much promise
Of fun times together

### quetzalplushie

Plumage of splendour
Green bird of plush--
(Enough of that, silly!
You're making me blush!)
